civihr_default_theme/tests/TemplateTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+class TemplateTest extends PHPUnit_Framework_TestCase {
```

Comment:
Thanks for adding this, but I'm afraid we'll have to keep the application without test coverage for this for now. This type of test is more suited for an e2e test suite, which, unfortunately, we still don't have for CiviHR. If we leave this test here, people will think this is the right way to write these tests and will start duplicating them. CiviHR will have e2e tests eventually (it has been discussed multiple times), and when that happens there will probably be test cases to cover this. For now, please remove the tests you've added.

Reply:
Your'e right, this test does suit e2e, at least for the happy case of ensuring the menu items appear. 

For edge cases (checking permissions etc.) I'd still say there's value in having unit tests. I don't think testing functions like this is the best way though. I would prefer not to require calling Drupal functions for a unit test, although it will require some bigger adjustments in this repo before we can use unit tests with mocking since there are no classes at all. I'll remove the tests for now anyway.